.travis.yml

```diff
@@ -17,11 +17,14 @@ env:
   - NJOBS=2
   # system is == 4.02.3
   - COMPILER="system"
+  # - COQ_GIT=git://scm.gforge.inria.fr/coq/coq.git
+  - COQ_GIT=https://github.com/ejgallego/coq.git
```

Comment:
Do you really want to still point to your personal repo once your PR has been merged on the Coq side?